new-lamassu-admin/src/pages/Analytics/Graph.js

```diff
@@ -0,0 +1,403 @@
+import BigNumber from 'bignumber.js'
+import * as d3 from 'd3'
+import React, { useCallback, useEffect, useMemo, useRef } from 'react'
+
+import {
+  java,
+  neon,
+  subheaderDarkColor,
+  offColor,
+  fontColor,
+  primaryColor,
+  fontSecondary
+} from 'src/styling/variables'
+
+const MINUTE = 60 * 1000
+const DAY = 24 * 60 * 60 * 1000
+const WEEK = 7 * 24 * 60 * 60 * 1000
+const MONTH = 30 * 24 * 60 * 60 * 1000
+
+const Graph = ({ data, representing, period, timezone }) => {
+  const ref = useRef(null)
+
+  const GRAPH_HEIGHT = 401
+  const GRAPH_WIDTH = 1163
+  const GRAPH_MARGIN = useMemo(
```

Comment:
Is there a reason for this to be a memoized react component? Why not a simple object?